Framework/DataHandling/inc/MantidDataHandling/LoadMuonNexusV2Helper.h

```diff
@@ -6,13 +6,21 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #pragma once
 #include "MantidAPI/WorkspaceGroup_fwd.h"
+#include "MantidAPI/Workspace_fwd.h"
+#include "MantidDataObjects/Workspace2D_fwd.h"
 #include "MantidGeometry/IDTypes.h"
 #include "MantidNexus/NexusClasses.h"
 
 #include <vector>
 
 namespace Mantid {
 namespace DataHandling {
+namespace MuonNexus {
+struct SampleInformation {
```

Comment:
Since this gets used in `MultiPeriodLoadMuonStrat` we should have it in it's own header, so people don't have to include the loader to pass around / use the struct

Also could we set some obvious values if this is uninitalized such as `-1` for temp and something not physically possible for magneticField 

I.e.
`double tempreature = -1`
So that if someone forgets to initalize it fails really obviously

Reply:
So outside of the loader this struct serves no purpose, as its just a glorified way to pack and unpack a nexus sample entry. I think keeping it here is best. As for initial values, while temperature is obvious if its in Kelvin, I'm not sure what I would put for magnetic field to signify that the value hasn't been set. Should I just add a constructor?